docs/tutorials/fundamentals/viewer.md

```diff
@@ -470,30 +473,39 @@ On the left side of the status bar there is a message about the position of the
-   * **Min** - minimum intensity projection. Similar to the maximum intensity projection, except that the minimum pixel value is used for the projected image instead of the maximum pixel value.
-   * **Std** - the standard deviation projection. At each pixel position, the standard deviation of the pixel intensities through the stack is the assigned value of that pixel position. Positions with large differences in the pixel intensities through the stack appear brighter in this projection.
-   * **Sum** - the sum projection simply adds together all the pixel values in the stack for a given position. In this projection, the image is typically re-scaled to a 16-bit image, as the sum of all the pixel intensity values usually exceeds 255, which would result in a completely white 8-bit image.
-   * **Mean** - the mean projection is the average intensity projection.  It simply averages all the pixel values in the stacks to make the final projected image.
-   * **Median** - the median projection takes the median pixel intensity for the final projected image.
+* **Copy scale and transforms** - copies the scale and transformation properties of the selected layer to the clipboard. This can be useful for pasting the properties to another layer. The following options are available in this sub-menu:
+  * **Copy all to clipboard** - copy all scale and transform properties to the clipboard.
+  * **Copy affine to clipboard** - copy the affine transformations to the clipboard.
+  * **Copy rotate to clipboard** - copy the rotation transformations to the clipboard.
+  * **Copy scale to clipboard** - copy the scale transformations to the clipboard.
+  * **Copy shear to clipboard** - copy the shear transformations to the clipboard.
+  * **Copy translate to clipboard** - copy the translation transformations to the clipboard.
+* **Apply scale/transforms from Clipboard** - applies the scale and translation properties from the clipboard to the selected layer.
+* **Duplicate Layer** - creates a second copy of the selected layer. Can be used on **Points**, **Shapes**, **Labels**, and **Image** layers. This is useful for testing your analysis on a copy instead of on the original image.
 * **Split RGB** - if the image layer is an RGB image, it will be split into 3 new layers with red, green, and blue values in separate layers.
 * **Split Stack** - if an image layer is a stack (has 3 or more dimensions), it is split into a list of layers along the axis. This option takes a little time to execute. Properties will be changed as follows:
     * **Colormap:** (magenta, green) for a stack with 2 channels, (CYMRGB) for stacks with more than 2 channels
     * **Blending:** additive
     * **Contrast_limits:** min and max values of the layer
     * All other properties, such as **Scale** and **Translate** will be propagated from the original stack.
```

Comment:
```suggestion
    * All other properties, such as **Scale** and **Translate** will be propagated from the original stack.
* *Merge to RGB** - combines a set of 3 image layers with the same dimensionality to a RGB layer. The layers must have one of `red`, `green`, and `blue` colormaps to indicate merging order.
```
Can we make up for me forgetting to document https://github.com/napari/napari/pull/7322 😬